ecmascript/regexp-lookbehind/alternations.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<!--
+Copyright (C) 2017 the V8 project authors. All rights reserved.
+This code is governed by the BSD license found in the LICENSE file.
+https://chromium.googlesource.com/v8/v8.git/+/main/LICENSE
+-->
+<title>JavaScript RegExp lookbehind assertions: alternations</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<script>
+test(t => {
+  // https://github.com/tc39/test262/blob/main/test/built-ins/RegExp/lookBehind/alternations.js
+  assert_array_equals("xabcd".match(/.*(?<=(..|...|....))(.*)/), ["xabcd", "cd", ""], "#1");
```

Comment:
Safari doesn’t yet support lookbehinds, and throws an early parse error when it encounters the first lookbehind in JS source. As a result, the test harness report page remains blank. Is this acceptable to convey the “fail” state?

(If we wanted the harness to show up, we’d have to turn all RegExp literals using lookbehinds into a string literal passed to the `RegExp()` constructor within a `try`-`catch`, with the `catch` handling the case of no parsing support.)